### PR TITLE
Update E2E build for 21.07

### LINKED ISSUE #96: Tests failing on "custom" end-to-end builds

### PR DESCRIPTION
**Pull request by wphicks**
Update E2E Dockerfile to accommodate 21.07 changes
Correct cp command that caused issues in main build
Update to GA TensorRT 8
Do not include libcudart in dependencies for both main and E2E build

Close #96

### CODE REVIEW

File: ops/e2e.Dockerfile [hunk 1]
```diff
@@ -133,12 +142,10 @@ RUN wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/nul
  && rm -rf /var/lib/apt/lists/*
 
 # Retrieve Triton server source
-ARG TRITON_VERSION=2.11.0
+ARG TRITON_VERSION=main
```
Comment (drobison00) [inline ops/e2e.Dockerfile:145]: Is this correct?  Not sure on triton versioning.

-- nevermind, it's the branch. I might suggest renaming TRITON_VERSION to TRITON_BRANCH

File: ops/e2e.Dockerfile [hunk 2]
```diff
@@ -147,6 +154,7 @@ RUN update-alternatives --install /usr/local/cuda cuda /usr/local/cuda-11.2 9 \
 # Build base Triton server
 ARG PARALLEL=4
 WORKDIR /src/server-${TRITON_VERSION}/build/output
+RUN patch -ruN -d /usr/include/ < ../libdcgm/dcgm_api_export.patch
```
Comment (drobison00) [inline ops/e2e.Dockerfile:157]: I dont see where the patch file is coming from,  is this added somewhere else?

**Review by drobison00 — APPROVED** [review]
Looks fine,  maybe a nit pick on the naming of version vs branch.

File: ops/e2e.Dockerfile (hunk 2 above)
Comment (wphicks) [reply ops/e2e.Dockerfile:157]: This is part of the Triton repo: https://github.com/triton-inference-server/server/blob/main/build/libdcgm/dcgm_api_export.patch

File: ops/e2e.Dockerfile (hunk 1 above)
Comment (wphicks) [reply ops/e2e.Dockerfile:145]: Yep! I like that much better. Clarifies the distinction between the release version and the branch name. I'll upgrade accordingly.

**Pull request merged**